2021-03-19 17:05:53,640 - test_order_manage.py[line:106] - ERROR: ---回单处理接口用例不通过，请检查问题---
2021-03-19 17:06:01,673 - test_order_manage.py[line:106] - ERROR: ---回单处理接口用例不通过，请检查问题---
2021-03-19 17:06:17,417 - test_order_manage.py[line:106] - ERROR: ---回单处理接口用例不通过，请检查问题---
2021-03-19 17:06:20,168 - test_order_manage.py[line:106] - ERROR: ---回单处理接口用例不通过，请检查问题---
2021-03-19 17:06:20,689 - test_order_manage.py[line:106] - ERROR: ---回单处理接口用例不通过，请检查问题---
2021-03-19 17:06:21,016 - test_order_manage.py[line:106] - ERROR: ---回单处理接口用例不通过，请检查问题---
2021-03-19 17:06:21,611 - test_order_manage.py[line:106] - ERROR: ---回单处理接口用例不通过，请检查问题---
2021-03-19 17:06:22,614 - test_order_manage.py[line:106] - ERROR: ---回单处理接口用例不通过，请检查问题---
2021-03-19 17:06:23,629 - test_order_manage.py[line:106] - ERROR: ---回单处理接口用例不通过，请检查问题---
2021-03-19 17:06:24,615 - test_order_manage.py[line:106] - ERROR: ---回单处理接口用例不通过，请检查问题---
2021-03-19 17:06:25,634 - test_order_manage.py[line:106] - ERROR: ---回单处理接口用例不通过，请检查问题---
2021-03-19 17:06:26,625 - test_order_manage.py[line:106] - ERROR: ---回单处理接口用例不通过，请检查问题---
2021-03-19 17:06:27,622 - test_order_manage.py[line:106] - ERROR: ---回单处理接口用例不通过，请检查问题---
2021-03-19 17:06:28,619 - test_order_manage.py[line:106] - ERROR: ---回单处理接口用例不通过，请检查问题---
2021-03-19 17:06:29,633 - test_order_manage.py[line:106] - ERROR: ---回单处理接口用例不通过，请检查问题---
2021-03-19 17:06:30,624 - test_order_manage.py[line:106] - ERROR: ---回单处理接口用例不通过，请检查问题---
2021-03-19 17:06:31,623 - test_order_manage.py[line:106] - ERROR: ---回单处理接口用例不通过，请检查问题---
2021-03-19 17:06:32,635 - test_order_manage.py[line:106] - ERROR: ---回单处理接口用例不通过，请检查问题---
2021-03-19 17:06:33,336 - test_order_manage.py[line:106] - ERROR: ---回单处理接口用例不通过，请检查问题---
2021-03-19 17:06:33,630 - test_order_manage.py[line:106] - ERROR: ---回单处理接口用例不通过，请检查问题---
2021-03-19 17:06:34,627 - test_order_manage.py[line:106] - ERROR: ---回单处理接口用例不通过，请检查问题---
2021-03-19 17:06:34,764 - test_order_manage.py[line:106] - ERROR: ---回单处理接口用例不通过，请检查问题---
2021-03-19 17:06:34,766 - test_order_manage.py[line:106] - ERROR: ---回单处理接口用例不通过，请检查问题---
2021-03-19 17:06:35,629 - test_order_manage.py[line:106] - ERROR: ---回单处理接口用例不通过，请检查问题---
2021-03-19 17:06:36,268 - test_order_manage.py[line:106] - ERROR: ---回单处理接口用例不通过，请检查问题---
2021-03-19 17:06:36,305 - test_order_manage.py[line:106] - ERROR: ---回单处理接口用例不通过，请检查问题---
2021-03-19 17:06:36,614 - test_order_manage.py[line:106] - ERROR: ---回单处理接口用例不通过，请检查问题---
2021-03-19 17:06:37,620 - test_order_manage.py[line:106] - ERROR: ---回单处理接口用例不通过，请检查问题---
2021-03-19 17:06:38,629 - test_order_manage.py[line:106] - ERROR: ---回单处理接口用例不通过，请检查问题---
2021-03-19 17:06:39,611 - test_order_manage.py[line:106] - ERROR: ---回单处理接口用例不通过，请检查问题---
2021-03-19 17:06:40,620 - test_order_manage.py[line:106] - ERROR: ---回单处理接口用例不通过，请检查问题---
2021-03-19 17:06:41,621 - test_order_manage.py[line:106] - ERROR: ---回单处理接口用例不通过，请检查问题---
2021-03-19 17:06:42,623 - test_order_manage.py[line:106] - ERROR: ---回单处理接口用例不通过，请检查问题---
2021-03-19 17:06:43,637 - test_order_manage.py[line:106] - ERROR: ---回单处理接口用例不通过，请检查问题---
2021-03-19 17:06:44,634 - test_order_manage.py[line:106] - ERROR: ---回单处理接口用例不通过，请检查问题---
2021-03-19 17:06:45,633 - test_order_manage.py[line:106] - ERROR: ---回单处理接口用例不通过，请检查问题---
2021-03-19 17:06:46,632 - test_order_manage.py[line:106] - ERROR: ---回单处理接口用例不通过，请检查问题---
2021-03-19 17:06:47,624 - test_order_manage.py[line:106] - ERROR: ---回单处理接口用例不通过，请检查问题---
2021-03-19 17:06:48,060 - test_order_manage.py[line:106] - ERROR: ---回单处理接口用例不通过，请检查问题---
2021-03-19 17:06:48,637 - test_order_manage.py[line:106] - ERROR: ---回单处理接口用例不通过，请检查问题---
2021-03-19 17:06:49,557 - test_order_manage.py[line:106] - ERROR: ---回单处理接口用例不通过，请检查问题---
2021-03-19 17:06:49,622 - test_order_manage.py[line:106] - ERROR: ---回单处理接口用例不通过，请检查问题---
2021-03-19 17:06:50,636 - test_order_manage.py[line:106] - ERROR: ---回单处理接口用例不通过，请检查问题---
2021-03-19 17:06:51,572 - test_order_manage.py[line:106] - ERROR: ---回单处理接口用例不通过，请检查问题---
2021-03-19 17:06:52,580 - test_order_manage.py[line:106] - ERROR: ---回单处理接口用例不通过，请检查问题---
2021-03-19 17:06:53,583 - test_order_manage.py[line:106] - ERROR: ---回单处理接口用例不通过，请检查问题---
2021-03-19 17:06:54,581 - test_order_manage.py[line:106] - ERROR: ---回单处理接口用例不通过，请检查问题---
2021-03-19 17:06:55,581 - test_order_manage.py[line:106] - ERROR: ---回单处理接口用例不通过，请检查问题---
2021-03-19 17:06:56,584 - test_order_manage.py[line:106] - ERROR: ---回单处理接口用例不通过，请检查问题---
2021-03-19 17:06:57,583 - test_order_manage.py[line:106] - ERROR: ---回单处理接口用例不通过，请检查问题---
2021-03-19 17:06:58,584 - test_order_manage.py[line:106] - ERROR: ---回单处理接口用例不通过，请检查问题---
2021-03-19 17:06:59,584 - test_order_manage.py[line:106] - ERROR: ---回单处理接口用例不通过，请检查问题---
2021-03-19 17:07:00,588 - test_order_manage.py[line:106] - ERROR: ---回单处理接口用例不通过，请检查问题---
2021-03-19 17:07:01,589 - test_order_manage.py[line:106] - ERROR: ---回单处理接口用例不通过，请检查问题---
2021-03-19 17:07:02,587 - test_order_manage.py[line:106] - ERROR: ---回单处理接口用例不通过，请检查问题---
2021-03-19 17:07:03,587 - test_order_manage.py[line:106] - ERROR: ---回单处理接口用例不通过，请检查问题---
2021-03-19 17:07:04,588 - test_order_manage.py[line:106] - ERROR: ---回单处理接口用例不通过，请检查问题---
2021-03-19 17:07:05,591 - test_order_manage.py[line:106] - ERROR: ---回单处理接口用例不通过，请检查问题---
2021-03-19 17:07:06,597 - test_order_manage.py[line:106] - ERROR: ---回单处理接口用例不通过，请检查问题---
2021-03-19 17:07:07,594 - test_order_manage.py[line:106] - ERROR: ---回单处理接口用例不通过，请检查问题---
2021-03-19 17:07:08,594 - test_order_manage.py[line:106] - ERROR: ---回单处理接口用例不通过，请检查问题---
2021-03-19 17:07:09,601 - test_order_manage.py[line:106] - ERROR: ---回单处理接口用例不通过，请检查问题---
2021-03-19 17:07:10,598 - test_order_manage.py[line:106] - ERROR: ---回单处理接口用例不通过，请检查问题---
2021-03-19 17:07:11,600 - test_order_manage.py[line:106] - ERROR: ---回单处理接口用例不通过，请检查问题---
2021-03-19 17:07:12,596 - test_order_manage.py[line:106] - ERROR: ---回单处理接口用例不通过，请检查问题---
2021-03-19 17:07:13,600 - test_order_manage.py[line:106] - ERROR: ---回单处理接口用例不通过，请检查问题---
2021-03-19 17:07:14,600 - test_order_manage.py[line:106] - ERROR: ---回单处理接口用例不通过，请检查问题---
2021-03-19 17:07:15,603 - test_order_manage.py[line:106] - ERROR: ---回单处理接口用例不通过，请检查问题---
2021-03-19 17:07:16,601 - test_order_manage.py[line:106] - ERROR: ---回单处理接口用例不通过，请检查问题---
2021-03-19 17:07:17,603 - test_order_manage.py[line:106] - ERROR: ---回单处理接口用例不通过，请检查问题---
2021-03-19 17:07:18,599 - test_order_manage.py[line:106] - ERROR: ---回单处理接口用例不通过，请检查问题---
2021-03-19 17:07:19,610 - test_order_manage.py[line:106] - ERROR: ---回单处理接口用例不通过，请检查问题---
2021-03-19 17:07:20,602 - test_order_manage.py[line:106] - ERROR: ---回单处理接口用例不通过，请检查问题---
2021-03-19 17:07:21,600 - test_order_manage.py[line:106] - ERROR: ---回单处理接口用例不通过，请检查问题---
2021-03-19 17:07:22,602 - test_order_manage.py[line:106] - ERROR: ---回单处理接口用例不通过，请检查问题---
2021-03-19 17:07:23,604 - test_order_manage.py[line:106] - ERROR: ---回单处理接口用例不通过，请检查问题---
2021-03-19 17:07:24,608 - test_order_manage.py[line:106] - ERROR: ---回单处理接口用例不通过，请检查问题---
2021-03-19 17:07:25,604 - test_order_manage.py[line:106] - ERROR: ---回单处理接口用例不通过，请检查问题---
2021-03-19 17:07:26,607 - test_order_manage.py[line:106] - ERROR: ---回单处理接口用例不通过，请检查问题---
2021-03-19 17:07:27,608 - test_order_manage.py[line:106] - ERROR: ---回单处理接口用例不通过，请检查问题---
2021-03-19 17:07:28,612 - test_order_manage.py[line:106] - ERROR: ---回单处理接口用例不通过，请检查问题---
2021-03-19 17:07:29,607 - test_order_manage.py[line:106] - ERROR: ---回单处理接口用例不通过，请检查问题---
